src/tool/hpcrun/sample_sources_registered.h

```diff
@@ -48,7 +48,7 @@
 #define REGISTERED_SAMPLE_SOURCES_H
 
 #include <sample-sources/sample_source_obj.h>
-#define MAX_POSSIBLE_SAMPLE_SOURCES 12
+#define MAX_POSSIBLE_SAMPLE_SOURCES 13
```

Comment:
Is it necessary?

Reply:
I am not sure. I added an entry for opencl in hpctoolkit/src/tool/hpcrun/sample-sources/ss-list.h which takes the total count from 12 to 13. @jmellorcrummey @Jokeren , I dont know the significance of MAX_POSSIBLE_SAMPLE_SOURCES, but is this change needed?

Reply:
please delete the #define MAX_POSSIBLE_SAMPLE_SOURCES rather than updating it. it is dead code.